### PR TITLE
Bluetooth: Mesh: Add sensor server to light_ctrl sample

### PR DESCRIPTION
**Pull request by omkar3141**
Adds a demonstration of usage of the sensor server model to report additional data about the device (in this case (dummy) energy use) to the light_ctrl sample.

### CODE REVIEW

**Review by mia-ko — CHANGES_REQUESTED** [review]

File: samples/bluetooth/mesh/light_ctrl/README.rst [hunk 1]
```diff
@@ -205,13 +227,27 @@ The default value of :kconfig:option:`CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_PROLONG`
 .. note::
    The configuration of light levels, fade time, and timeouts can be changed by altering the configuration parameters in the :file:`prj.conf` file, and rebuilding the sample.
 
+Configure the Sensor Server model on the **Mesh Light Fixture** node:
+
+* Bind the model to **Application Key 1**, make sure to bind the same application key to the peer Sensor Client.
```
Comment (akredalen) [inline samples/bluetooth/mesh/light_ctrl/README.rst:232]:
```suggestion
* Bind the model to **Application Key 1**. Make sure to bind the same application key to the peer Sensor Client.
```

**Review by akredalen — CHANGES_REQUESTED** [review]
Looks good to me :) Just a few nitpicks.

File: samples/bluetooth/mesh/light_ctrl/README.rst [hunk 2]
```diff
@@ -205,13 +227,27 @@ The default value of :kconfig:option:`CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_PROLONG`
...
+
+The Sensor Server model is now configured and able to receive messages from and send data to the peer Sensor Client.
+
+Configure the Sensor Setup Server model on the **Mesh Sensor** node:
+
+* Bind the model to **Application Key 1**. make sure to bind the same application key to the peer Sensor Client.
+
```
Comment (Balaklaka) [inline samples/bluetooth/mesh/light_ctrl/README.rst:239]: Should it say something about Publish/Subscriptions before it is able to receive sensor settings and it is configured? Same for sensor server.

**Review by Balaklaka — COMMENTED** [review]
Should we extend the sensor_client sample to support this functionality?

**Review by akredalen — APPROVED** [review]

**Review by mia-ko — APPROVED** [review]

**Pull request merged**